### ISSUE TITLE
Empty Exception Body

### ISSUE THREAD

**Issue by OKTAYKIR** [Research-Institute/json-api-dotnet-core#143]
I am going to test bad request (400) with a customized error message. The test gets 400 as status code; however the content body is empty.

```
[HttpPost]
public override async Task<IActionResult> PostAsync([FromBody] Contract.User entity)
{
if (entity == null)
{
    return Error(new Error() { Status  = "400", Title = "Invalid request body", Detail = "The provided request 
    entity body is not valid." });
}

    return await base.PostAsync(entity);
}
```

**Comment by jaredcnance** [comment]
@OKTAYKIR  thanks for reporting the issue. A couple things that will help me solve this faster:

- What version are you on?
- Are you certain 400 is not being thrown from this call `base.PostAsync(entity);`
- If you have time, can you fork this repo and reproduce the issue in a test? Looks like the [current tests](https://github.com/Research-Institute/json-api-dotnet-core/blob/master/test/UnitTests/Controllers/JsonApiControllerMixin_Tests.cs#L48-L54) are only making assertions about the status code

**Comment by OKTAYKIR** [comment]
* I use version 2.1.0. 
* I am certain because, i debugged this case.
* If I can find time, I will do it.

Thanks.

**Comment by jaredcnance** [comment]
@OKTAYKIR  Copying your code and running it gives me the following result:

```json
{
    "errors": [
        {
            "title": "Invalid request body",
            "detail": "The provided request entity body is not valid.",
            "status": "400"
        }
    ]
}
```
Here's the cURL request:
```
curl -X POST \
  http://localhost:5000/api/v1/people \
  -H 'accept: application/vnd.api+json' \
  -H 'content-type: application/vnd.api+json' \
```

Can you create a repository with your code that reproduces the issue?

**Comment by OKTAYKIR** [comment]
Hi @jaredcnance ,

I created repository with sample code that reproduces the issue.

https://github.com/OKTAYKIR/jsonapidotnetcore.example

**Comment by jaredcnance** [comment]
@OKTAYKIR thanks for taking the time to report and reproduce this issue. It should be resolved in v2.1.1.

If you're still having trouble please reopen this issue. For your reference, see #145 for the fix.

Thanks
